src/openrct2/platform/shared.c

```diff
@@ -146,6 +146,13 @@ void platform_refresh_video(bool recreate_window)
     gfx_invalidate_screen();
 }
 
+static uint32 _ticks = 0;
+
+void platform_advance_ticks()
+{
+    _ticks += 25;
```

Comment:
ms to advance ticks by, integer.

25 => 1000/25 = 40fps
16.67 => 1000/16.67 = 60fps
16 => 1000/16 = 62.5fps
17 => 1000/17 = 58.82fps
8 => 1000/8 = 125fps